setmyenv/README.md

```diff
@@ -0,0 +1,8 @@
+# SetMyEnvironment
+Sets up original environment for Ubuntu 16.04.
```

Comment:
I'm not sure what you mean by original? 
Maybe you mean "development" or "standalone" ?